Scripts/card_comment_gen.py

```
@@ -90,28 +92,76 @@ def setCommentGen(target_set, target_id):
     NeutralNonCollect = []
     NoneClass = []
 
-    all = [Heroes, HeroPowers, Druid, DruidNonCollect, Hunter, HunterNonCollect, Mage,
-           MageNonCollect, Paladin, PaladinNonCollect, Priest, PriestNonCollect,
-           Rogue, RogueNonCollect, Shaman, ShamanNonCollect, Warlock, WarlockNonCollect,
-           Warrior, WarriorNonCollect, DemonHunter, DemonHunterNonCollect, Dual, DualNonCollect,
-           Neutral, NeutralNonCollect, NoneClass]
+    all = [
```

Ruff (0.15.21)

[error] Variable `all` is shadowing a Python builtin

(A001)